WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -160,15 +169,30 @@ private extension GetStartedViewController {
         setTableViewMargins(forWidth: view.frame.width)
     }
 
-    func setupContinueButton() {
-        let tableFooter = UIView(frame: Constants.footerFrame)
-        tableFooter.addSubview(continueButton)
-        tableFooter.pinSubviewToSafeArea(continueButton, insets: Constants.footerButtonInsets)
-        continueButton.translatesAutoresizingMaskIntoConstraints = false
-        continueButton.isEnabled = false
-        continueButton.addTarget(self, action: #selector(handleSubmitButtonTapped(_:)), for: .touchUpInside)
-        continueButton.accessibilityIdentifier = "Get Started Email Continue Button"
-        tableView.tableFooterView = tableFooter
+    func setupTableFooterView() {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.spacing = Constants.FooterStackView.spacing
+        stackView.layoutMargins = Constants.FooterStackView.layoutMargins
+        stackView.isLayoutMarginsRelativeArrangement = true
+
+        stackView.addArrangedSubview(continueButton)
+
+        if WordPressAuthenticator.shared.configuration.whatIsWPComURL != nil {
+            let stackViewWithCenterAlignment = UIStackView()
+            stackViewWithCenterAlignment.axis = .vertical
+            stackViewWithCenterAlignment.alignment = .center
```

Comment:
nit: just curious, what are the reasons for using a stack view for the button?

Reply:
Used a stack view with center alignment to avoid the `What is WordPress.com` button from occupying the whole width. i.e. To avoid `What is WordPress.com` button having the same width as `Continue` button. 

Kindly share if you have any suggestions about this. 😄

Reply:
Yes, that will work as well. I would like to keep the stack view usage as I think this reads better as you mentioned. 😄